datastream-server-restli/src/main/java/com/linkedin/datastream/server/dms/ZookeeperBackedDatastreamStore.java

```diff
@@ -38,7 +38,8 @@
 public class ZookeeperBackedDatastreamStore implements DatastreamStore {
 
   private static final Logger LOG = LoggerFactory.getLogger(ZookeeperBackedDatastreamStore.class.getName());
-
+  private static final Double ONE_MEBIBYTE = (double) (1024 * 1024);
+  private static final Double ZNODE_BLOB_SIZE_LIMIT = ONE_MEBIBYTE;
```

Comment:
Do you really need 2 variables?

Reply:
The purpose of these variables are little different
- ZNODE_BLOB_SIZE_LIMIT ––> Validation of data size per znode based on the znode limit (=1MB) 
- ONE_MEBIBYTE  --> Converting bytes to MB to get encoded data size in MBs. 

Ideally, I can use the same variable, but I kept them both for better understanding purposes of two different operations happening on the same value.